sympy/matrices/expressions/factorizations.py

```diff
@@ -6,29 +6,29 @@ class Factorization(MatrixExpr):
     shape = property(lambda self: self.arg.shape)  # type: ignore
 
 class LofLU(Factorization):
-    predicates = Q.lower_triangular,
+    predicates = property(lambda self: (Q.lower_triangular,))
 class UofLU(Factorization):
-    predicates = Q.upper_triangular,
+    predicates = property(lambda self: (Q.upper_triangular,))
```

Comment:
The normal way to use the property decorator is like:
```
@property
def predicates(self):
    return (Q.upper_triangular,)
```
It would be better if we didn't have code that messed around with `property` like this in the codebase.